src/backend/metal/src/command.rs

```diff
@@ -721,7 +722,6 @@ impl Journal {
     }
 
     fn record(&self, command_buf: &metal::CommandBufferRef) {
-        let _ap = AutoreleasePool::new(); // for encoder creation
```

Comment:
This removal seems ok but maybe we should indicate that this is handled by https://github.com/gfx-rs/gfx/pull/2267/files#diff-abc29e1de3205709a964ff5e3b0e6333R1494 instead

Reply:
I don't know the best way for us to enforce the autorelease pools, tbh. Just trying to put comments doesn't protect us against a breakage during refactor, or from using a method accidentally that requires a pool but we don't have one...